.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "Signal_iQ"]
```

Comment:
do you have submodules? Let's do the following:
1. in readme capture what this depends on.
2. as part of the script simply checkout those projects since you need them as part of the built.

Would that work?